csss-site/src/events/templates/events/fall_hacks2020.html

```diff
@@ -0,0 +1,13 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+	 <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Computing Science Student Society Fall Hacks 2020 submissions<a>

```

Comment:
Please add a header tag for the string `Computing Science Student Society Fall Hack 2020`

and then below it just update the link to `submissions`

Reply:
@SpartanApple, once this is fixed, I think I will merge